test/runTest.ts

```diff
@@ -13,9 +13,9 @@ async function main() {
         const extensionTestsPath = path.resolve(__dirname, './');
 
         // Download VS Code, unzip it and run the integration test
-        await runTests({version: '1.44.1', extensionDevelopmentPath, extensionTestsPath, launchArgs: [path.resolve(__dirname, '../../test/resources/sampleNodeApp/')] });
+        await runTests({extensionDevelopmentPath, extensionTestsPath, launchArgs: [path.resolve(__dirname, '../../test/resources/sampleNodeApp/')] });
```

Comment:
Is it possible that not defining a version may create a unknown failure of test cases in future?
Or Mocha will take care of it?

Reply:
Nope, it wasn't new enough to run the latest webpacked js., so just removed the version pinning. I will find a version which works for case. It is a good catch 👍

Reply:
1.46.1 seems to be working.

Reply:
Okay, Cool :)